Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/ukhomeofficedigital/nodejs-base:v6
```

Comment:
Ancient....

Reply:
don't you think it's a little out of scope? but ok I'll update